Determine branch name for downgraded version when updating with "--downgrade" option (#165)
truckersmp-cli/truckersmp-cli: README.md

(Not available)|`--activate-native-d3dcompiler-47`|Activate native 64-bit `d3dcompiler_47.dll` when starting (Needed for D3D11 renderer)
(Not available)|`--check-windows-steam`|Check for the Windows Steam version on updating when using Proton
(Not available)|`--disable-proton-overlay`|Disable Steam Overlay when using Proton

> (Not available)|`--downgrade`|Downgrade to latest TruckersMP-compatible version when updating Note: This option implies "--update" option and is ignored if "--beta" ("-b") option is specified

Reviewer: ```suggestion
(Not available)|`--downgrade`|Downgrade to the latest version that is supported by TruckersMP. Note: This option implies "--update" option and is ignored if "--beta" ("-b") option is specified
```

$ truckersmp-cli --ats --start
```

#### How to downgrade games

> When the latest game version is not compatible with TruckersMP yet, if user updates game with `--downgrade` option (that implies `--update`), truckersmp-cli installs the latest TruckersMP-compatible version of game by determining Steam game branch name, using [TruckersMP Web API][truckersmp:webapi].

Reviewer: ```suggestion
When the latest game version is not compatible with TruckersMP yet, the user can downgrade the games with `--downgrade` (that implies `--update`) which installs the latest version that is supported by TruckersMP by determining the Steam game branch name by using [TruckersMP Web API][truckersmp:webapi].
```
